spec/models/mailing_spec.rb

```diff
@@ -1,26 +1,40 @@
 require 'spec_helper'
 
 describe Mailing do
-  let(:mailing) { Mailing.new(from: 'from@email.com', to: Role::ROLES,
+  let!(:mailing) { Mailing.new(from: 'from@email.com', to: Role::ROLES,
     cc: 'cc@email.com', bcc: 'bcc@email.com', subject: 'subject', body: '# body') }
 
   it { is_expected.to have_many(:submissions).dependent(:destroy) }
 
   describe '#sent?' do
-    skip 'TODO'
+
+    it 'false before submission' do
+      expect(mailing.subject).to eq 'subject'
+      mailing.save!
```

Comment:
What's the sake of calling `save` here once again after using a `let!` in the beginning? Does it do anything new?